src/components/Club/cashoutSection.tsx

```
@@ -0,0 +1,21 @@
+
+export function CashoutSection() {
+    return (
+        <div className="relative flex min-h-[72vh] flex-col items-center justify-center overflow-x-hidden bg-pink-1">
+            <motion.img
+                src={CashoutNow.src}
+                className="h-auto max-w-[90%] cursor-pointer"
+                alt="Cashout Now"
+                initial={{ opacity: 0, translateY: 20 }}
+                whileInView={{ opacity: 1, translateY: 0 }}
+                transition={{ type: 'spring', damping: 10 }}
+                onClick={() => (window.location.href = '/cashout')}
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)